exonum-java-binding/service-archetype/pom.xml

```diff
@@ -43,6 +51,11 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-archetype-plugin</artifactId>
           <version>${maven-archetype.version}</version>
+          <configuration>
+            <properties>
+              <nativeLibPath>${project.parent.basedir}/core/rust/target/debug</nativeLibPath>
```

Comment:
@dip56 Would you please test that release builds continue to work (we didn't use to use the native library in archetype ITs), and the generated project works with the installed app (the current 0.7.0-SNAPSHOT)?